main.go

```diff
@@ -124,16 +142,124 @@ func NewExporter(server string, timeout time.Duration) *Exporter {
 		),
 		evictions: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "items_evicted_total"),
-			"Number of valid items removed from cache to free memory for new items.",
+			"Total number of valid items removed from cache to free memory for new items.",
 			nil,
 			nil,
 		),
 		reclaimed: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "items_reclaimed_total"),
-			"Number of times an entry was stored using memory from an expired entry.",
+			"Total number of times an entry was stored using memory from an expired entry.",
 			nil,
 			nil,
 		),
+		malloced: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "malloced_bytes"),
+			"Number of bytes of memory allocated to slab pages.",
+			nil,
+			nil,
+		),
+		itemsNumber: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "slab", "items_number"),
```

Comment:
Let's use `memcached_slab_current_items` to be consistent with the global metric name and avoid confusion with the slab number.